x/oracle/keeper/validatorWhiteList.go

```diff
@@ -5,32 +5,32 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func (k Keeper) SetOracleWhiteList(ctx sdk.Context, validatorList []sdk.ValAddress) {
+func (k Keeper) SetOracleWhiteList(ctx sdk.Context, networkDescriptor types.NetworkDescriptor, validatorList []sdk.ValAddress) {
 	store := ctx.KVStore(k.storeKey)
-	key := types.WhiteListValidatorPrefix
+	key := networkDescriptor.GetPrefix()
 	store.Set(key, k.Cdc.MustMarshalBinaryBare(validatorList))
 }
 
-func (k Keeper) ExistsOracleWhiteList(ctx sdk.Context) bool {
-	key := types.WhiteListValidatorPrefix
+func (k Keeper) ExistsOracleWhiteList(ctx sdk.Context, networkDescriptor types.NetworkDescriptor) bool {
+	key := networkDescriptor.GetPrefix()
 	return k.Exists(ctx, key)
 }
 
 //
-func (k Keeper) GetOracleWhiteList(ctx sdk.Context) (valList []sdk.ValAddress) {
+func (k Keeper) GetOracleWhiteList(ctx sdk.Context, networkDescriptor types.NetworkDescriptor) (valList []sdk.ValAddress) {
```

Comment:
instead of mapping a network descriptor to an array of addresses, let's map a network descriptor to an address and map that address to a uint that is that users power.